middleware.js

```diff
@@ -198,12 +197,14 @@ module.exports = function(options) {
 
       mkdirp(dirname(cssPath), '0700', function(err) {
         if (err) {
-          return error(err);
+          error(err);
+          cssDone = true;
+          return doneWriting();
```

Comment:
Should we add an option to toggle propagation to next middleware in case of error? My concern is someone might have taken dependency on this behavior by now and this would break them.